### ISSUE TITLE
bs-select messes up tab order (accessibility)

### ISSUE THREAD

**Issue by DmitryEfimenko** [mgcrea/angular-strap#1318]
I'm using ngAria to create accessibility attributes. I see that these attributes are created correctly. However, when pressing `Tab`, once the selected item becomes the `bs-select` element, the tab order cannot get passed that element even though there are other elements waiting to be selected.

See [Plunker example](http://plnkr.co/edit/3a6MtyNkQI6WMADp1V5p?p=preview).

**Comment by JereStay** [comment]
+1

**Comment by mattklein999** [comment]
+1

**Comment by adam-marshall** [comment]
+1

**Comment by DmitryEfimenko** [comment]
Either I misunderstand something or the proposed fix does not fix the issue.

What actually fixes the issue is commenting out lines [184-185](https://github.com/mgcrea/angular-strap/blob/master/src/select/select.js#L184-L185). Though I'm not sure what that'll break.

**Comment by vmlf01** [comment]
The fix simply hides the select list element so you can tab out of the field into the next one.
